bitrise.yml

```diff
@@ -2272,6 +2272,150 @@ workflows:
             focus-ios/focus-ios-tests/tools/sentry-cli --auth-token "$SENTRY_AUTH_TOKEN" upload-dif \
               --org mozilla --project klar-ios "$BITRISE_DSYM_DIR_PATH"
 
+# RelPro workflows
+  release_promotion_promote:
+    before_run:
+    - SPM_Common_Steps
+    steps:
+    - script@1.2.1:
+        title: Set staging environment variables
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+            envman add --key BITRISE_SCHEME --value FirefoxStaging
+            envman add --key EXPORT_METHOD --value development
+        run_if: '{{envcontain "GIT_REPOSITORY_URL" "staging"}}'
```

Comment:
Maybe we should match this with the `run_if` from the push workflow?

`run_if: {{enveq "BITRISE_APP_SLUG" "staging-firefox-ios"}}`